zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -1599,8 +1599,10 @@ public void processPacket(ServerCnxn cnxn, ByteBuffer incomingBuffer) throws IOE
                 }
             }
             if (authReturn == KeeperException.Code.OK) {
-                LOG.debug("Authentication succeeded for scheme: {}", scheme);
-                LOG.info("auth success {}", cnxn.getRemoteSocketAddress());
+                ZooKeeperSaslServer saslServer = cnxn.zooKeeperSaslServer;
```

Comment:
Almost… but I'm afraid you still have a leftover `saslServer` variable here :)

Reply:
Sorry about that! Fixed it now.

Reply:
No problem!  LGTM.